README.md

```diff
@@ -1,87 +1,86 @@
-libdns - Universal DNS provider APIs for Go
-===========================================
+# Godaddy for `libdns`
 
-<a href="https://pkg.go.dev/github.com/libdns/libdns"><img src="https://img.shields.io/badge/godoc-reference-blue.svg"></a>
-
-**⚠️ Work-in-progress. Exported APIs are subject to change.**
-
-`libdns` is a collection of free-range DNS provider client implementations written in Go! With libdns packages, your Go program can manage DNS records across any supported providers. A "provider" is a service or program that manages a DNS zone.
-
-This repository defines the core interfaces that provider packages should implement. They are small and idiomatic Go interfaces with well-defined semantics.
-
-The interfaces include:
-
-- [`RecordGetter`](https://pkg.go.dev/github.com/libdns/libdns#RecordGetter) to list records.
-- [`RecordAppender`](https://pkg.go.dev/github.com/libdns/libdns#RecordAppender) to append new records.
-- [`RecordSetter`](https://pkg.go.dev/github.com/libdns/libdns#RecordSetter) to set (create or change existing) records.
-- [`RecordDeleter`](https://pkg.go.dev/github.com/libdns/libdns#RecordDeleter) to delete records.
-
-[See full godoc for detailed documentation.](https://pkg.go.dev/github.com/libdns/libdns)
+[![godoc reference](https://img.shields.io/badge/godoc-reference-blue.svg)](https://pkg.go.dev/github.com/artknight/libdns-godaddy)
 
+This package implements the libdns interfaces for the [Godaddy API](https://developer.godaddy.com/doc/endpoint/domains)
 
 ## Example
 
-To work with DNS records managed by Cloudflare, for example, we can use [libdns/cloudflare](https://pkg.go.dev/github.com/libdns/cloudflare):
+Here's a minimal example of how to get all your DNS records using this `libdns` provider (see `_example/main.go`)
```

Comment:
You _can_ repeat this example in the readme, but you have an example .go file, I'd recommend just using one or the other.